service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -148,4 +153,32 @@ void chainCallsTogether() throws Exception {
         newRunCaptor.getValue().submissionTimestamp(),
         greaterThan(OffsetDateTime.now().minus(Duration.ofSeconds(60))));
   }
+
+  @Test
+  void tooManyRecordIds() throws Exception {
```

Comment:
Also worth checking "exactly the right number of record IDs"?

Reply:
easy enough! though I won't check all the stuff that happens after the request is successfully returned -- that should be in the ticket that follows this one